dsn_test.go

```diff
@@ -105,3 +105,12 @@ func TestDSNV0Failed(t *testing.T) {
 	runDSNTestFail(t, "jdbc://user:yury_db@db_name")
 	runDSNTestFail(t, "firebolt://yury_db@dn_name?account_name=firebolt_account")
 }
+
+func TestDSNCoreHappyPath(t *testing.T) {
```

Comment:
Let's add failure cases as well - when `url` is provided, but some of the other parameters are missing.